src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -17,6 +17,21 @@ export default (accessToken, userID, opts) => {
     identityVerificationLoader: gravityLoader(
       (id) => `identity_verification/${id}`
     ),
+    identityVerificationsLoader: gravityLoader(
+      "identity_verifications",
+      {},
+      { headers: true }
+    ),
+    identityVerificationOverridesLoader: gravityLoader(
```

Comment:
This and the references don't need the `headers: true` , that's just used to resolve w/ the headers since the expectation is a `total_count` query param will be passed and so that count header is desired. But these two loaders don't seem to be used in that way (although I suppose there's no harm in keeping it with `headers: true`, more of an FYI).

Reply:
I couldn't get this to work. Always received `null` back for the lists when declaring loaders without `headers: true`. Any ideas?